jetty-start/src/main/java/org/eclipse/jetty/start/CommandLineBuilder.java

```diff
@@ -169,23 +230,9 @@ public String toString(String delim)
      *
      * @return the toString but each arg that has spaces is surrounded by {@code '} (single-quote tick)
      */
-    public String toQuotedString()
+    public String toCommandLine()
```

Comment:
Another public method that cannot be renamed in Jetty 10.
Good for Jetty 12 tho.